### ISSUE TITLE
Support Restricted Mode

### ISSUE THREAD

**Issue by fenhl** [shardulm94/vscode-trailingspaces#67]
Since this extension does not execute code from the workspace as far as I can tell, it should be allowed in Restricted Mode.

**Comment by shardulm94** [comment]
You're right — this extension only reads document text and applies a regex to strip trailing whitespace; it never executes workspace code or shells out, so there's no reason for it to be disabled in Restricted Mode.

This has been addressed. The manifest now declares Workspace Trust support (and virtual workspace support) via [`capabilities.untrustedWorkspaces`](https://github.com/shardulm94/vscode-trailingspaces/blob/master/package.json):

```json
"capabilities": {
    "untrustedWorkspaces": {
        "supported": true
    },
    "virtualWorkspaces": true
}
```

This landed in #87 and shipped in **v0.4.3**, so the extension now runs normally in Restricted Mode (and in virtual workspaces such as github.dev). Closing as completed — thanks for the report!
